packages/core/signal.mjs

```diff
@@ -86,6 +86,27 @@ export const tri2 = fastcat(isaw2, saw2);
 
 export const time = signal(id);
 
+/**
+ *  The mouse's x or y position value ranges from 0 to 1.
+ *
+ * @return {Pattern}
+ * @example
+ * n(mousex.segment(4).range(0,7)).scale("C:minor")
+ * n(mousey.segment(4).range(0,7)).scale("C:minor")
+ *
+ */
+let _mouseY = 0,
+  _mouseX = 0;
+if (typeof window !== 'undefined') {
+  document.onmousemove = (e) => {
```

Comment:
maybe do `document.addEventListener('click', () => {})` here, because onmousemove might get overwritten?

Reply:
If i do so, the value only change when i click. Shouldn't a signal constantly be changed?

Reply:
sry I mistyped, I meant `mousemove` instead of `click`